doc/api.rst

```diff
@@ -2,11 +2,32 @@
 API
 ###
 
-:py:mod:`mne_connectivity`:
```

Comment:
This will make it so that this no longer works:

```
:mod:`mne_connectivity`
```
So it should be restored. You probably also need to restore the `automodule`. If you don't like the text that gets displayed (from `__init__.py`, I think), you can remove it (I think?) or at least make it better.

Reply:
Added it back in.